auto_nag/scripts/configs/tools.json

```diff
@@ -229,6 +229,11 @@
     {
         "months_lookup": 24
     },
+    "several_dups":
+    {
+        "weeks_lookup": 3,
+        "number_dups": 3
+    },
```

Comment:
@calixteman do we need to add a `receivers` with `release-mgmt@mozilla.com`, or is it default?

Reply:
yes we do.